docs/blog/biz/README.md

```diff
@@ -0,0 +1,45 @@
+ # Anomalib
```

Comment:
I think `docs/blog/biz/README.md` is not ready to be merged. Instead, we should only merge the technical blog post.